backends/advanced/src/advanced_omi_backend/workers/memory_jobs.py

```
@@ -281,4 +283,23 @@ def enqueue_memory_processing(
+        
+        default_queue.enqueue(
+            generate_title_summary_job,
+            conversation_id,
+            job_timeout=300,
+            result_ttl=JOB_RESULT_TTL,
+            job_id=summary_job_id,
+            description=f"Generate title and summary for conversation {conversation_id[:8]}",
+        )
+        logger.info(f"📥 RQ: Enqueued summary job {summary_job_id} for conversation {conversation_id}")
+    except Exception as e:
+        logger.error(f"Failed to enqueue summary job: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)